src/cellRenderers/CellProvider.js

```diff
@@ -234,10 +245,23 @@ var CellProvider = Base.extend('CellProvider', {
         config.minWidth = config.minWidth + 2 * (iconWidth);
     },
 
-    renderMultiLineText: function(gc, x, y, height, width, config, val) {
+    /**
+     * @summary Renders multiline text
+     * @param {CanvasGraphicsContext} gc
+     * @param {number} config.bounds.x - the x screen coordinate of my origin
```

Comment:
Since `config.bounds` is a `Rectangle` just say:
```javascript
/**
 * @param {Rectangle} config.bounds - The bounding rect of the cell to render.
 */
```
Note however that Rectangle is not in this project. (It is in the "rectangular" npm module). So in this case, add it to ./src/jsdocs/external-types.js as a @typedef with a @see link to it's jsdoc page (http://openfin.github.io/rectangular/Rectangle.html). This will make Rectangle a known type to jsdoc throughout the app.